packages/plans-grid/src/plans-table/index.tsx

```diff
@@ -23,6 +23,10 @@ export interface Props {
 	currentDomain?: DomainSuggestions.DomainSuggestion;
 	disabledPlans?: { [ planSlug: string ]: string };
 	locale: string;
+	showTaglines?: boolean;
+	CTAVariation?: 'FULL_WIDTH' | 'NORMAL';
```

Comment:
We should export these values from the package similar to how domain picker has done [here](https://github.com/Automattic/wp-calypso/blob/master/packages/domain-picker/src/domain-picker/suggestion-item.tsx#L22-L23) so we can use it like [this](https://github.com/Automattic/wp-calypso/pull/47318/files#diff-a88807ae898bfacf7689a16c723644ee8d5ddf40677fd0ce119761a404d0b897R10)